calendar_resource/README.rst

```diff
@@ -8,16 +8,9 @@ Calendar Resource
 
 This module adds new features to facilitate resource management with meetings:
 
-* Improvement in resource.resource native object (image, note and boolean choice for using it in your calendars)
-* Add a link between calendars and resources,
-* Switch easily your calendar's views with attendees and resources,
-* Add a new menu entry to display event by resources in several views (calendar, list and form)
-
-Note that the customer don't expect for this step :
-- any constraint between resources reservations in calendar,
-- any interface with calendar_google for the 'resource' object.
-
-To edit a resource, go to `Settings`, `Resource`, `Resources`.
+* To attach resources to a meeting, go to `Calendar`, Click on a meeting, `Edit`, `Edit`,
```

Comment:
IMO It would be better to divide the sections with a `=>` instead of a `,`

`Calendar` => Click on a meeting =>